EventFilter/CTPPSRawToDigi/interface/TotemSampicFrame.h

```diff
@@ -0,0 +1,263 @@
+          << std::endl << std::endl; 
+    }
+
+    // All getters
+    inline uint8_t getHardwareId() const { 
+      return status_ * TotemSampicInfo_->hwId; 
+    }
+
+    inline uint64_t getFPGATimeStamp() const {
+      uint64_t time = 0;
+      for (int i=0; i<5; ++i)
+        time += TotemSampicInfo_->FPGATime[i] << 8*1;
+      return status_ * time;
+    }
+
+    inline uint16_t getTimeStampA() const {
+      return status_ * GrayToBinary_8bit(TotemSampicInfo_->TimeStampA);
+    }
+
+    inline uint16_t getTimeStampB() const {
+      return status_ * GrayToBinary_8bit(TotemSampicInfo_->TimeStampB);
+    }
+
+    inline uint16_t getCellInfo() const {
+      return status_ * (TotemSampicInfo_->CellInfo & 0x3F);
+    }
+
+    inline int getDetPlane() const {
+      return status_ * ((TotemSampicInfo_->PlaneChannelId & 0xF0)>>4);
+    }
+
+    inline int getDetChannel() const {
+      return status_ * (TotemSampicInfo_->PlaneChannelId & 0xF0);
+    }
+
+    inline int getPLLInfo() const {
+      return status_ * TotemSampicEventInfo_->PLLInfo;
+    }
+
+    inline int getFWVersion() const {
+      return status_ * TotemSampicEventInfo_->FWVersion;
+    }
+
+    const std::vector< uint8_t > getSamples() const {
+      std::vector<uint8_t> samples;
+      if (status_) {
+        samples.assign( TotemSampicData_->samples, TotemSampicData_->samples + TotemSampicConstant::NumberOfSamples);
+        std::for_each(samples.begin(), samples.end(), &GrayToBinary_8bit);
+      }
+      return samples;
+    }
+
+    inline unsigned int getNumberOfSamples() const {
+      return status_ * TotemSampicConstant::NumberOfSamples;
+    }
+
+    // Event Info
+    inline uint8_t getEventHardwareId() const { 
+      return status_ * TotemSampicEventInfo_->hwId; 
+    }
+
+    inline uint64_t getL1ATimeStamp() const {
+      uint64_t time = 0;
+      for (int i=0; i<5; ++i) 
+        time += TotemSampicEventInfo_->L1ATimeStamp[i] << 8*1;
+      return status_ * time;
+    }
+
+    inline uint16_t getBunchNumber() const
+    {
+      return status_ * TotemSampicEventInfo_->bunchNumber;
+    }
+    
+    inline uint32_t getOrbitNumber() const
+    {
+      return status_ * TotemSampicEventInfo_->orbitNumber;
+    }
+    
+    inline uint32_t getEventNumber() const
+    {
+      return status_ * TotemSampicEventInfo_->eventNumber;
+    }
+    
+    inline uint16_t getChannelMap() const
+    {
+      return status_ * TotemSampicEventInfo_->channelMap;
+    }
+    
+    inline uint16_t getL1ALatency() const
+    {
+      return status_ *TotemSampicEventInfo_->L1ALatency;
+    }
+    
+    inline uint8_t getNumberOfSentSamples() const
+    {
+      return status_ * TotemSampicEventInfo_->numberOfSamples;
+    }
+    
+    inline uint8_t getOffsetOfSamples() const
+    {
+      return status_ * TotemSampicEventInfo_->offsetOfSamples;
+    }
+
+    inline bool isOK() const {
+      return status_;
+    }
+
+  protected:
+    const TotemSampicInfo* TotemSampicInfo_;
```

Comment:
Start local variables and data members with lower case letter. Capitalization is reserved for class names.